### PR TITLE
Inference support predictor stream : add gpu resources.

### PR DESCRIPTION
**Pull request by jiweibo**
### PR types

Others
### PR changes

Others
### Describe

add gpu resources.

### CODE REVIEW

**Review by hp03 — COMMENTED** [review]
LGTM

**Review by hp03 — APPROVED** [review]
LGTM

File: paddle/phi/backends/gpu/gpu_context.cc [hunk 1]
```diff
@@ -202,27 +203,65 @@ struct GPUContext::Impl {
...
+                           &max_threads_per_mp_,
+                           &max_threads_per_block_,
+                           &max_grid_dim_size_);
+    phi::InitStream(&stream_);
     InitEigenDevice();
-    InitBlasHandle();
-    InitBlasLtHandle();
-    InitDNNHandle();
-    InitSolverHandle();
-    InitSparseHandle();
```
Comment (shangzhizhou) [inline paddle/phi/backends/gpu/gpu_context.cc:212]: gpu_context的修改，请训练框架同学double check一下

**Review by shangzhizhou — APPROVED** [review]
LGTM

**Review by chenwhql — APPROVED** [review]
LGTM

**Pull request merged**